run/idp-sql/test/retry.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+##
+# retry.sh
```

Comment:
I like this script. This alone might be worth breaking out to a gist and socializing.